src/mrpro/operators/_SensitivityOp.py

```diff
@@ -0,0 +1,23 @@
+"""Class for Sensitivity Operator."""
+
+import torch
+
+from mrpro.data import CsmData
+
+
+class SensitivityOp:  # todo: later put (LinearOp) here
+    """Sensitivity operator class."""
+
+    def __init__(self, csm: CsmData):
+        self.C = csm.data
+
+    def forward(self, u: torch.Tensor):
+        """Apply the forward operator, thus expand the num_coils dimension."""
```

Comment:
Change the argument name to "img_data" for all methods
A more detailed docstring including the shape of the tensor would be nice.